frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -1142,6 +1137,36 @@ class Worksheet extends React.Component {
                     .show();
             }.bind(this),
         });
+    };
+
+    deleteThisWorksheet() {
+        // TODO: put all worksheet dialogs into WorksheetDialogs.js if possible
+        let deleteWorksheetDialog = (
+            <Dialog
+                open={true}
+                onClose={this.toggleBundleBulkMessageDialog}
+                aria-labelledby='delete-worksheet-confirmation-title'
+                aria-describedby='delete-worksheet-confirmation-description'
+            >
+                <DialogTitle id='delete-worksheet-confirmation-title'>
+                    {'WARNING: Deleted worksheets cannot be recovered!'}
```

Comment:
Maybe tone this language down a little -- warning can just be non-capitalized and replace ! with a period

Reply:
Since it's a title I'm leaving the period out